### DISCUSSION TITLE
Installed a new version duckdb==0.8.0 | TransactionContext Error: Failed to commit: mutex lock failed: Invalid argument

### DISCUSSION THREAD

**Discussion by Starlord22** [duckdb/duckdb#7628] [Q&A]
1. Installed a new version of Duckdb (pip install duckdb==0.8.0)
2. Trying to connect DuckDB to DBeaver.  (Earlier version 0.7.0 was working fine.)
3. Now getting 'TransactionContext Error: Failed to commit: mutex lock failed: Invalid argument'

How to address this. I am new to Duckdb and Databases in general. Is there a Discord server where community users can ask/seek more active help on run time errors?

**Comment by Mytherin** [comment]
The Discord server can be found at [discord.duckdb.org](https://discord.duckdb.org)
